### PR TITLE
EWSO365 - handle attachments with non utf-8 encoded chars

### PR DESCRIPTION
**Pull request by Itay4**
## Status
- [x] Ready

## Description
handle non utf-8 encoded chars in attachments 

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests

### CODE REVIEW

**Comment by Itay4** [comment]
see failing test in https://app.circleci.com/pipelines/github/demisto/content/88082/workflows/cb0d59a4-0a1d-415c-9e38-58d359b3579f/jobs/364682

File: Packs/EWS/ReleaseNotes/1_8_18.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### EWS O365
+- Fixed an issue where the integration failed to parse attachment with non UTF-8 encoded characters.
```
Comment (bakatzir) [inline Packs/EWS/ReleaseNotes/1_8_18.md:4]:
```suggestion
- Fixed an issue where parsing attachments with non UTF-8 encoded characters failed.
```

**Review by yaakovi — APPROVED** [review]
is there an issue?

**Comment by Itay4** [comment]
> is there an issue?

not yet

**Pull request merged**